src/styles/blocks/month.scss

```diff
@@ -0,0 +1,35 @@
+.month {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  gap: $dayBlockGap;
+  flex-wrap: wrap;
+
+  width: ($dayBlockSize + $dayBlockGap) * $daysInWeek + $monthPadding * 2 - 1;
+  height: ($dayBlockSize + $dayBlockGap) * $weekRowsInMonth + $monthPadding * 2
+    - 1;
```

Comment:
You may consider defining variable for the width and height and perform this calculation there. 

Also just suggestion that wrap your multiplication in a pair of round brackets for better readability.